recipes/genie/all/test_v1_package/scripts/genie.lua

```diff
@@ -0,0 +1,23 @@
+newoption {
```

Comment:
please, re-use from test package with relative path and remove this copied file.